infrastructure/create-github-environment.js

```diff
@@ -14,38 +14,30 @@ const args = minimist(process.argv.slice(2), {
 const config = {
   environment: args.environment,
   dockerhub: {
-    ORGANISATION: 'opencrvs', // This may be a dockerhub organisation or the same as the username
-    REPOSITORY: 'ocrvs-farajaland',
+    ORGANISATION: 'opencrvs', // EDIT: This should be your dockerhub organisation
+    REPOSITORY: 'ocrvs-farajaland', // EDIT: This should be your dockerhub, private repository
     USERNAME: process.env.DOCKER_USERNAME,
     TOKEN: process.env.DOCKER_TOKEN
   },
   github_repository: {
-    ORGANISATION: 'opencrvs',
-    REPOSITORY_NAME: 'opencrvs-farajaland'
+    ORGANISATION: 'opencrvs', // EDIT: This should be your github organisation name
+    REPOSITORY_NAME: 'opencrvs-farajaland' // EDIT: This should be your github forked countryconfig repository name
   },
   ssh: {
-    SSH_HOST: process.env.SSH_HOST, // IP address for the manager
-    SSH_USER: process.env.SSH_USER,
-    SSH_KEY: process.env.SSH_KEY
+    SSH_HOST: process.env.SSH_HOST, // Note: For "prodution" environment server clusters of (2, 3 or 5 replicas) this is always the IP address for just 1 manager server
+    SSH_USER: 'provision',
+    SSH_KEY: process.env.SSH_KEY,
+    SSH_ARGS: `-J jump@${process.env.VPN_HOST_ADDRESS}`
```

Comment:
Note that this wouldn't be needed for non-production environments. Could we also have it as a variable instead of a secret as it doesn't have anything secret in it and it would make debugging a lot easier?